examples/whiteboard/README.md

```diff
@@ -0,0 +1,24 @@
+# Fastify WebSocket Whiteboard
```

Comment:
please inspire from the other readmes and show instructions about how to run this in cloud run too. you must be able to run the whole app in cloud run, meaning, serve the static assets from there, for istance using fastify-static.

Reply:
okey.. i wasn't aware of that. i will look how to serve static content using fastify

Reply:
this repo is accompanying the article about fastify on google cloud run, so all examples should be able to be deployed and run there